pyproject.toml

```diff
@@ -113,6 +113,7 @@ select = [
 ]
 ignore = [
     "N806", # non-lowercase variable (we use N* for axes lengths)
+    "N802", # non-lowercase function name (we use N* for axes lengths)
```

Comment:
I've typically used either in-line `# noqa N802` or added something to the `[tool.ruff.lint.per-file-ignores]` section for this, mostly to limit the situations in which we ignore this rule.